saleor/order/tests/test_calculations.py

```diff
@@ -1141,3 +1142,74 @@ def test_order_undiscounted_total(mocked_fetch_order_prices_if_expired):
     assert undiscounted_total == quantize_price(
         expected_undiscounted_total, order.currency
     )
+
+
+@patch("saleor.plugins.avatax.plugin.AvataxPlugin")
+@patch("saleor.order.calculations._recalculate_with_plugins")
+@patch("saleor.plugins.manager.PluginsManager.get_taxes_for_checkout")
+@override_settings(PLUGINS=["saleor.plugins.avatax.plugin.AvataxPlugin"])
```

Comment:
The test should not be strictly related to AvataxPlugin. If you want to test single tax plugin call, you should mock PluginsManager, and confirm that the request was passed to single plugin (or just confirm that proper method for calling the method on single plugin was called)